app/src/main/java/org/openedx/app/OpenEdXApp.kt

```diff
@@ -31,7 +35,8 @@ class OpenEdXApp : Application() {
                 .setGcmSenderId(org.openedx.core.BuildConfig.FIREBASE_GCM_SENDER_ID)
                 .build()
             Firebase.initialize(this, options)
+            //Add Firebase Analytics for tracking
+            analytics.addAnalyticsTracker(FirebaseAnalytics(context = baseContext))
```

Comment:
We can move this within the `init` block of AnalyticsManager as well. 

So that all configuration related to analytics is kept within the AnalyticsManager class.